src/main/java/com/example/online_shop_api/Controller/EmployeeController.java

```diff
@@ -35,9 +33,9 @@ public ResponseEntity<?> registerNewEmployee(@RequestBody @Valid EmployeeRequest
         return employeeService.registerNewEmployee(employeeRequestDto, bindingResult);
     }
 
-    @GetMapping("/profile")
-    public ResponseEntity<MyUserDetails> showProfile(Authentication authentication) {
-        return employeeService.showEmployeeProfile(authentication);
-    }
+//    @GetMapping("/profile")
```

Comment:
we can throw away the things we don't use

Reply:
This is temporarily commented out code until the security is applied.

Reply:
We will change this method. Its return type will be ResponseEntity<EmployeeResponseDto>. It shouldn't return MyUserDetails since it contains password and other sensitive data that will not be displayed.